bitnami/argo-cd/Chart.yaml

```
@@ -38,5 +38,5 @@ maintainers:
     url: https://github.com/bitnami/charts
 name: argo-cd
 sources:
-  - https://github.com/bitnami/charts/tree/main/bitnami/argo-cd
-version: 6.6.9
+- https://github.com/bitnami/charts/tree/main/bitnami/argo-cd
+version: 6.7.0
```

Please bump the patch version instead:
```suggestion
version: 6.6.10
```

---

Seems logical. Done!